docs/source/config_file.rst

```diff
@@ -965,6 +996,10 @@ of your repo (or append it to the end of an existing ``pyproject.toml`` file) an
     python_version = "2.7"
     warn_return_any = true
     warn_unused_configs = true
+    exclude = [
+        '^file1\.py$',  # TOML single-quoted string (no escaping necessary)
+        "^file2\\.py$",  # TOML double-quoted string (backslash needs escaping)
```

Comment:
```suggestion
        '^file1\.py$',  # TOML literal string (no escaping necessary)
        "^file2\\.py$",  # TOML basic string (backslash and other characters need escaping)
```

I think it might be helpful to use the terms from the TOML spec, to make searching things like "TOML multline literal string" return more consistent results.

Reply:
Let's not assume the reader has knowledge of the TOML spec.  The distinction between single-quotes vs. double-quotes was lost on me until this last edit.  I would not want us to drop the explicit statement of single vs. double.

Reply:
I agree with cclauss here! :-)

Reply:
> The distinction between single-quotes vs. double-quotes was lost on me until the last edit.

Both the [first version](https://github.com/python/mypy/pull/11828/files/bbd5431551d39059c03d6fde72ea8c2686ef0440#r774334354) and the [second version](https://github.com/python/mypy/pull/11828/files/bbd5431551d39059c03d6fde72ea8c2686ef0440#r774334225) use the terms "single-quotes" and  "double-quotes", so the distinction would be in other parts of the comment.

> I would not want us to drop the explicit statement of single vs. double.

The examples the comments reference contain the `'` and `"` characters, so the terms "single-quotes" and "double-quotes" are duplicate information.

> Let's not assume the reader has knowledge of the TOML spec.

If the reader has knowledge of the TOML spec, the comments would not add any information for them.

I'm always in favor of using searchable terms in documentation that does not link to the information it's referencing.

This is a compromise:

```suggestion
        '^file1\.py$',  # TOML literal string (single-quotes, no escaping necessary)
        "^file2\\.py$",  # TOML basic string (double-quotes, backslash and other characters need escaping)
```

But this is a very minor thing, all told.

Reply:
Sure, accepted!